com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -172,23 +174,37 @@ internal int GetTopMostLitLayer()
 
         internal void UpdateMesh()
         {
-            var bounds = default(Bounds);
             switch (m_LightType)
             {
                 case LightType.Freeform:
-                    bounds = LightUtility.GenerateShapeMesh(m_Mesh, m_ShapePath, m_ShapeLightFalloffSize);
+                    m_LocalBounds = LightUtility.GenerateShapeMesh(m_Mesh, m_ShapePath, m_ShapeLightFalloffSize);
                     break;
                 case LightType.Parametric:
-                    bounds = LightUtility.GenerateParametricMesh(m_Mesh, m_ShapeLightParametricRadius, m_ShapeLightFalloffSize, m_ShapeLightParametricAngleOffset, m_ShapeLightParametricSides);
+                    m_LocalBounds = LightUtility.GenerateParametricMesh(m_Mesh, m_ShapeLightParametricRadius, m_ShapeLightFalloffSize, m_ShapeLightParametricAngleOffset, m_ShapeLightParametricSides);
                     break;
                 case LightType.Sprite:
-                    bounds = LightUtility.GenerateSpriteMesh(m_Mesh, spriteLightCookie);
+                    m_LocalBounds = LightUtility.GenerateSpriteMesh(m_Mesh, spriteLightCookie);
                     break;
                 case LightType.Point:
-                    bounds = LightUtility.GenerateParametricMesh(m_Mesh, 1.412135f, 0, 0, 4);
+                    m_LocalBounds = LightUtility.GenerateParametricMesh(m_Mesh, 1.412135f, 0, 0, 4);
                     break;
             }
-            boundingSphereRadius = isShapeLight ? CalculateBoundingSphereRadius(bounds) : m_PointLightOuterRadius;
+        }
+
+        private void UpdateBoundingSphere()
+        {
+            if (!isShapeLight)
```

Comment:
change `isShapeLight` to something that actually what it means... `isPointLight` because everywhere it is used, it is negated...

Reply:
Fixed.